### DISCUSSION TITLE
3mhz on uv k5

### DISCUSSION THREAD

**Discussion by Firdaus-gh** [egzumer/uv-k5-firmware-custom#173] [General]
Is it possible to lower the frequency UV K5 up to 3mhz?

**Comment by ErikS-web** [comment]
Here you can see a [blockdiagram of the radio](https://github.com/ludwich66/Quansheng_UV-K5_Wiki/wiki/Block_Diagram)

The heart of the radio is the IC BK4819.
Look up the specifications (datasheet) and you can read that the answer is `NO`.

**Comment by f5gkw** [comment]
🤣🤣🤣🤣

**Comment by kamilsss655** [comment]
@Firdaus-gh technically no, but you can use an up-converter (i.e https://support.nooelec.com/hc/en-us/articles/360005812474-Ham-It-Up-Upconverter) to receive frequencies that are lower than radio can normally receive.

I can confirm it works.

@egzumer It would be great to have a setting 'RxOffset' that would allow to enter frequencies without having to do math. I think fagci and ivj already does that with 'Offset' settings. Currently if using 40Mhz up-converter in order to receive 3Mhz one has to enter VFO freq to 43Mhz.  It would be awesome if one could set '40Mhz' offset in settings and the radio would display '3Mhz' receive frequency while actually listening on 43Mhz for example.

Also implementations of fagci and ivj are a bit flawed and can be improved because even with 40Mhz offset, one cannot set frequency to lower than 18Mhz, even though one should be able to (considering 40Mhz offset).

Fagci and ivj  also have 'upconverter' setting with hardcoded VFOs like 50Mhz and 100Mhz i think. This solution is also not great because there are bunch of upconverters that have different frequencies, so user should be able to set 'offset' as a number, also allowing for possible frequency deviations per unit (+/- khz).

Thus 'RxOffset' setting would be ideal.

**Reply by ErikS-web** [reply]
The UVK5 radio is intended for 50 - 600 mHz and does not receive optimal on other frequenties.

Effort is useless. Better buy a specific HF tranciever/receiver to enjoy HAM
